controller/work-item-link-blackbox_test.go

```diff
@@ -264,7 +264,7 @@ func CreateWorkItemLinkCategory(name string) *app.CreateWorkItemLinkCategoryPayl
 }
 
 // CreateWorkItem defines a work item link
-func CreateWorkItem(workItemType string, title string) *app.CreateWorkitemPayload {
+func CreateWorkItem(workItemType satoriuuid.UUID, title string) *app.CreateWorkitemPayload {
```

Comment:
why `satoriuuid`  and not `uuid` as everywhere else in the code ? Or shoudn't we use `satoriuuid` everywhere for a matter of consistency ?